tcping.go

```diff
@@ -103,7 +103,12 @@ func usage() {
 	colorYellow("\n[optional flags]\n")
 
 	flag.VisitAll(func(f *flag.Flag) {
-		colorYellow("  -%s : %s\n", f.Name, f.Usage)
+		flagName := f.Name
+		if len(f.Name) > 1 {
+			flagName = "-" + flagName
+		}
+
+		colorYellow("  -%s : %s\n", flagName, f.Usage)
```

Comment:
Explanation on this: go's flag package by default uses one dash for flags, but for convenience two dashes are also supported and they're simply equivalent of 1-dash commands (`-j`==`--j`, `-pretty`==`--pretty`).

This extra check is here to add additional dash for commands that are longer than 1 character, because that's usually how it's being done in most (of not all) CLIs
<img width="856" alt="image" src="https://github.com/pouriyajamshidi/tcping/assets/5007271/7d06cfe8-9ec9-4bef-ac8a-250f2c66a9d3">

Reply:
Good job.